packages/sdk-components-react/src/form.ws.ts

```diff
@@ -20,7 +20,6 @@ export const meta: WsComponentMeta = {
   category: "forms",
   type: "container",
   invalidAncestors: ["Form", "Button", "Link"],
-  label: "Form",
```

Comment:
oops
<img width="282" alt="image" src="https://github.com/user-attachments/assets/b5fbab4a-de0a-462b-b6f1-4458bebcffac">

Reply:
Shit, somehow missed this one